spec/api/user_spec.rb

```diff
@@ -149,6 +149,16 @@ def parse_response_and_verify_counts(expected_threads, expected_comments)
              verify_counts_multiple_groups(2, 6, "100", "3000")
              verify_counts_multiple_groups(1, 5, "100", "8")
           end
+
+          context "standalone threads" do
+            before(:each) do
+              make_standalone_thread_with_comments(@users['u100'])
+            end
+
+            it 'does not return standalone thread or comments in counts' do
+              verify_counts(1, 5, "100")
```

Comment:
I'm having a hard time telling where these numbers are coming from. I realize that's a fault of the helper methods available to you here, but could you add a comment like the one on line 96?

`# "setup_10_threads" creates 1 thread ("t0") and 5 comments (in "t0") authored by user 100.`